Databricks 3 (#2384)
fishtown-analytics/docs.getdbt.com: website/docs/docs/get-started/connect-your-database.md

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/bq_oauth_as_user.gif" title="Link Button in dbt Cloud Credentials Screen" />

## Connecting to Databricks

> We have two supported adapters to connect to Databricks: [dbt-databricks](/connect-your-database#dbt-databricks) and [dbt-spark](/connect-your-database#dbt-spark). For accounts on dbt 1.0 or later, we recommend using the dbt-databricks adapter. The dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks.

Reviewer: @schottj I'm curious if the dbt-spark adapter will now be maintained by the community? We might want to link to the [verified adapters](https://docs.getdbt.com/docs/supported-data-platforms#verified-adapters) section of the supported data platforms page. It looks like both are verified, is that not correct anymore? 

It sounds like we really want people to use the new one. We might want to add it to this table and remove the old one: 
https://docs.getdbt.com/docs/supported-data-platforms#verified-adapters

Reviewer 2: dbt-spark will continue to be maintained by dbt Labs.

We do want to recommend people to use the new one, yes.

> We have two supported adapters to connect to Databricks: [dbt-databricks](/connect-your-database#dbt-databricks) and [dbt-spark](/connect-your-database#dbt-spark). For accounts on dbt 1.0 or later, we recommend using the dbt-databricks adapter. The dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks.

Reviewer 2: ```suggestion
You can connect to Databricks by using one of two supported adapters: [dbt-databricks](/connect-your-database#dbt-databricks) and [dbt-spark](/connect-your-database#dbt-spark). For accounts on dbt 1.0 or later, we recommend using the dbt-databricks adapter. The dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks. Connecting to Databricks via dbt-spark will be deprecated in the future.
```
Adding a comment on the eventual deprecation of using dbt-spark to Databricks.

### dbt-databricks Adapter
dbt-databricks is compatible with the following versions of dbt Core in dbt Cloud with varying degrees of functionality.

| Feature | dbt Versions |
| ----- | ----------- | 
| dbt-databricks | Available starting with dbt 1.0 in dbt Cloud|
| Unity Catalog | Available starting with dbt 1.1 | 
| Python models | Available starting with dbt 1.3 |

The dbt-databricks adapter offers:
- **Easier set up**
- **Better defaults:**

> The dbt-databricks is more opinionated, guiding users to an improved experience with less effort. Design choices of the dbt-databricks adapter include defaulting to Delta format, using merge for incremental models, and running expensive queries with Photon.

Reviewer: "more opinionated" than dbt-spark adapter?

Reviewer 2: Yes :)
By default it uses the Databricks specific features that are not part of Spark.

> The dbt-databricks is more opinionated, guiding users to an improved experience with less effort. Design choices of the dbt-databricks adapter include defaulting to Delta format, using merge for incremental models, and running expensive queries with Photon.

Reviewer 2: ```suggestion
The dbt-databricks adapter is more opinionated, guiding users to an improved experience with less effort. Design choices of this adapter include defaulting to Delta format, using merge for incremental models, and running expensive queries with Photon.
```

- **Support for Unity Catalog:**
Unity Catalog allows Databricks users to centrally manage all data assets, simplifying access management and improving search and query performance. Databricks users can now get three-part data hierarchies – catalog, schema, model name – which solves a longstanding friction point in data organization and governance.

To set up the Databricks connection, supply the following fields:

| Field | Description | Examples |
| ----- | ----------- | -------- |
| Server Hostname | The hostname of the Databricks account to connect to | dbc-a2c61234-1234.cloud.databricks.com |
| HTTP Path | The HTTP path of the Databricks cluster or SQL warehouse | /sql/1.0/warehouses/1a23b4596cd7e8fg |
| Catalog | Name of Databricks Catalog (optional) | Production |

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/dbt-databricks.png" title="Configuring a Databricks connection using the dbt-databricks adapter"/>

### dbt-spark Adapter

> dbt Cloud supports connecting to Databricks using

Reviewer: ```suggestion
The dbt Spark adapter connects dbt Cloud to Databricks using a `Cluster` or
`Endpoint` configuration:
```

Reviewer 2: Should it be `The dbt-spark adapter` instead?

Reviewer 2: Actually I think the current wording flows better. Just my opinion though, please revert if you really like the other one better ;)
